packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -109,6 +107,10 @@ const ImageGridItem = styled.div`
   }
 `;
 
+const ImageContentsWrap = styled.div`
```

Comment:
I'm being a bit picky here but, can this be either `ImageContentsWrapper` or `ImageContent`?

Reply:
done :)